### PR TITLE
Add auto-tooltips.js module

### PR DESCRIPTION
**Pull request by nitriques**

### CODE REVIEW

File: src/modules/auto-tooltips.js [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+	var win = $(window);
+	var site = $('#site');
+	var sitePages = $('#site-pages');
+	var page = $('.page');
+	var ajaxTooltipsUrl = '/' + $('html').attr('lang') + '/ajax-tooltips/';
+	
+	var INITED_CLASS = 'is-tooltips-inited';
+	var TOOLTIPS_SELECTOR = '.js-tooltips';
+	//var TOOLTIPS_ITEM_SELECTOR = '.js-tooltips-item';
```
Comment (fhamon) [inline src/modules/auto-tooltips.js]: ( ͡° ͜ʖ ͡°)

File: src/modules/auto-tooltips.js [hunk 2]
```diff
@@ -0,0 +1,322 @@
+/**
+ * @author Deux Huit Huit
+ *
+ * Auto tooltips module
+ *
+ * Scan certain places for words and inject a tooltip in place of the text word
+ *
+ */
+/* jshint maxstatements:38 */
```
[the remark this answers is not in the document]
Comment (nitriques) [reply src/modules/auto-tooltips.js]: ahaha yeah

[the remark this answers is not in the document]
Comment (nitriques) [reply src/modules/auto-tooltips.js]: Not needed we are ok ;)

File: src/modules/auto-tooltips.js (hunk 1 above)
Comment (nitriques) [reply src/modules/auto-tooltips.js]: Done

**Pull request merged**